lib.sh

```diff
@@ -631,18 +631,20 @@ fi
 ram_check() {
 install_if_not bc
 mem_available="$(awk '/MemTotal/{print $2}' /proc/meminfo)"
-mem_available_gb="$(echo "scale=2; $mem_available/(1024*1024)" | bc)"
+mem_available_gb="$(printf '%0.2f\n' $(echo "scale=3; $mem_available/(1024*1024)" | bc))"
```

Comment:
Why this change if I may ask?

Reply:
It adds a leading zero, so instead of 

> .94GB installed, 2GB required

you get

> 0.94GB installed, 2GB required

which more closely aligns the formatting between the two numbers. When looking the rendered whiptail dialog, it was a bit more readable with the leading zero when quickly mentally comparing `0.94` and `2`

![Screen Shot 2020-05-04 at 15 31 29](https://user-images.githubusercontent.com/5077610/81010738-597fd600-8e1c-11ea-8cbd-a88d67a1e52f.png)

Reply:
OK, tested now and it looks good as far as I've tested.